.circleci/config.yml

```
@@ -116,6 +116,9 @@ jobs:
           no_output_timeout: 25m
           command: |
             export PYTHONPATH=$PWD/build-install/lib/python3.11/site-packages
+            # TODO: remove jupyterlite-sphinx pin when
+            # gh-21212 is resolved upstream
+            python -m pip install jupyterlite-sphinx==0.15.0
```

```suggestion
            python -m pip install jupyterlite-sphinx>=0.16.2
```

This should work now that 0.16.2 including the fix is in PyPI. Thanks!

---

I guess since `pyproject.toml` (and `requirements/doc.txt` used by the CI here) contain `jupyterlite-sphinx>=0.13.1`, we could fix CI by just closing/ignoring this PR.

To be more correct we might want update `pyproject.toml` with your new version requirements and then use the dep propagation script to update the `requirements` files.

---

OK that may be safer just to be sure we don't install the wrong versions there. @steppi does that sound good?

---

I went ahead and did that, I can't think of anything better.